File: i2f-springboot/i2f-springboot-ops-starter/src/main/java/i2f/springboot/ops/common/OpsSecureCertPair.java
package i2f.springboot.ops.common;

import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author Ice2Faith
 * @date 2025/11/2 12:40
 * @desc
 */
@Data
@NoArgsConstructor
public class OpsSecureCertPair {
    protected String serverCert;
    protected String clientCert;
    public OpsSecureCertPair(String serverCert, String clientCert) {
        this.serverCert = serverCert;
        this.clientCert = clientCert;
    }
}

File: i2f-springboot/i2f-springboot-ops-starter/src/main/resources/assets/ops/common/OpsSecureCertPair.js
/**
 *
 * @param serverCert {string}
 * @param clientCert {string}
 * @type {OpsSecureCertPair}
 * @constructor {OpsSecureCertPair}
 */
function OpsSecureCertPair(serverCert,clientCert) {
    /**
     * @type {string}
     */
    this.serverCert=serverCert;
    /**
     * @type {string}
     */
    this.clientCert=clientCert;
}